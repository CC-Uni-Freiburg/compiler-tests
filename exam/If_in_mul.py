print(69 * (10 if 4 > 0 else 6)) # 690
print(69 * (10 if 4 < 0 else 6)) # 414
print((69 if 4 > 0 else 7) * 5) # 345
print((69 if 4 < 0 else 7) * 5) # 35
print((69 if True else 7) * (69 if True else 7)) # 4761
print((69 if True else 7) * (69 if False else 7)) # 483
print((69 if False else 7) * (69 if True else 7)) # 483
print((69 if False else 7) * (69 if False else 7)) # 49
