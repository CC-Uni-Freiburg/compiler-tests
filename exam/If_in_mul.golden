69041434535476148348349
